Standards/scs-XXXX-v1-volume-backup-service.md

```diff
@@ -0,0 +1,71 @@
+Thus, an arbitrary OpenStack cloud may or may not offer this feature.
+
+This standard aims to make this functionality the default in SCS clouds so that customers can expect the feature to be usable.
+
+## Design Considerations
+
+The standard should make sure that the feature is available and usable but should not limit the exact implementation (e.g. choice of backend driver) any further than necessary.
+
+### Options considered
+
+#### Only recommend volume backup feature, use images as alternative
+
+As an alternative to the volume backup feature of the Block Storage API, Glance images can also be created based on volumes and act as a backup under certain circumstances.
+As an option, this standard could keep the actual integration of the volume backup feature optional and guide users how to use images as backup targets instead in case the feature is unavailable.
+
+However, it is not guaranteed that the Glance backend storage is separate from the volume storage.
+For instance, both could be using the same Ceph cluster.
+In such case, the images would not count as genuine backups.
+
+Although users are able to download images and transfer them to a different storage location, this approach might also prove unfeasible depending on the image size and the existence (or lack) of appropriate target storage on the user side.
+
+Furthermore, incremental backups are not possible when creating Glance images from volumes either.
+This results in time-consuming backup operations of fully copying a volume everytime a backup is created.
+
+#### Mandate the availability of the volume backup feature
+
+This option is pretty straightforward.
+It would make the volume backup feature mandatory for SCS clouds.
+This way users can expect the feature to be available and usable.
+
+With this, users can leverage functionalities like incremental backups and benefit from optimized performance of the backup process due to the tight integration with the volume service.
+
+## Decision
+
+This standard decides to go with the second option and makes the volume backup feature mandatory in the following way:
+
+In an SCS cloud, the volume backup functionality MUST be configured properly and its API as defined per `/v3/{project_id}/backups` MUST be offered to customers.
+If using Cinder, a suitable [backup driver](https://docs.openstack.org/cinder/latest/configuration/block-storage/backup-drivers.html) MUST be set up.
+
+The volume backup target storage SHOULD be a separate storage system from the one used for volumes themselves.
+
+## Related Documents
+
+- [OpenStack Block Storage v3 Backup API reference](https://docs.openstack.org/api-ref/block-storage/v3/index.html#backups-backups)
+
+## Conformance Tests
+
+Conformance tests include using the `/v3/{project_id}/backups` Block Storage API endpoint to create a volume and a backup of it as a non-admin user and subsequently restore the backup on a new volume while verifying the success of each operation.
```

Comment:
This does only check MUST-requirementa of this standard. But we have a SHOULD-requirement to: Operating a separate storage backend for backup. As we can not test this automatically, check for additional storage backend is not part of python script. However, we should mention that separate storage backends are not validated in conformance tests in section "Conformance Tests" in order to be transparent for SCS customers. Furthermore, I suggest to create a new issue to collect all manual tests and to decide to proceed with manual audits.

Reply:
> Furthermore, I suggest to create a new issue to collect all manual tests and to decide to proceed with manual audits.

Good point. It has been coming up a lot recently, so I took the initiative:

https://github.com/SovereignCloudStack/issues/issues/668

Reply:
I added a note to the Conformance Tests section.